eng/common/scripts/Test-SampleMetadata.ps1

```diff
@@ -247,6 +247,7 @@ begin {
         "azure-network-watcher",
         "azure-notebooks",
         "azure-notification-hubs",
+        "azure-openai",
```

Comment:
@weshaggard Is fine if I add this new slug directly in java repo

Reply:
These new slugs need to be added in the azure-sdk-tools repo because eng/common is shared.